<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest" tests="3" time="212.769" ignored="0" failures="0" timestamp="11 Apr 2018 10:58:25 GMT" skipped="1" hostname="DESKTOP-NM05SVP" errors="2">
  <testcase name="deletePortFolioTest" time="0.000" classname="com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest">
    <skipped/>
  </testcase> <!-- deletePortFolioTest -->
  <testcase name="createPortFolioTest" time="69.849" classname="com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest">
    <error type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.qtpselenium.rediff.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:47)
	at com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest.createPortFolioTest(PortFolioTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.AssertionError: The following asserts failed:
	Values Dont match. Got value as portfolio1801
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:348)
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.waitTillSelectionToBe(GenericKeywords.java:269)
	at com.qtpselenium.rediff.hybrid.keywords.AppKeywords.verifyPortFolio(AppKeywords.java:75)
	... 26 more
]]>
    </error>
  </testcase> <!-- createPortFolioTest -->
  <testcase name="createPortFolioTest" time="142.920" classname="com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest">
    <error type="java.lang.reflect.InvocationTargetException">
      <![CDATA[java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.qtpselenium.rediff.hybrid.driver.DriverScript.executeKeywords(DriverScript.java:47)
	at com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest.createPortFolioTest(PortFolioTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:421)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.AssertionError: The following asserts failed:
	Values Dont match. Got value as portfolio1801
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.reportFailure(GenericKeywords.java:348)
	at com.qtpselenium.rediff.hybrid.keywords.GenericKeywords.waitTillSelectionToBe(GenericKeywords.java:269)
	at com.qtpselenium.rediff.hybrid.keywords.AppKeywords.verifyPortFolio(AppKeywords.java:75)
	... 26 more
]]>
    </error>
  </testcase> <!-- createPortFolioTest -->
</testsuite> <!-- com.qtpselenium.rediff.hybrid.testcases.portfoliosuite.PortFolioTest -->
